docs/manual/user_guide.adoc

```diff
@@ -163,11 +163,7 @@ Result  pass
 
 === Result Interpretation
 
-HTML Results
```

Comment:
You can just suggest to open the `html` file in favorite browser.

Reply:
Ok. So I'll put back the sections and write this recommendation.